SPECS/luajit/luajit.spec

```diff
@@ -0,0 +1,261 @@
+%global rctag beta3
+
+Summary:        Just-In-Time Compiler for Lua
+Name:           luajit
+Version:        2.1.0
+%global apiver %(v=%{version}; echo ${v%.${v#[0-9].[0-9].}})
+%global srcver %{version}%{?rctag:-%{rctag}}
+Release:        0.26%{?rctag:%{rctag}}%{?dist}
```

Comment:
Clarification:

I am not aware of this convention of release; double checking if you think this is the `Right Thing` ?

Reply:
Our tooling expects Release tag should be in [number][dist] format. Updated SPEC to match to our tooling and put details in change log.